scripts/sv/run_whole_pipeline.sh

```diff
@@ -43,12 +43,11 @@ SV_ARGS=${*:-${SV_ARGS:-""}}
 # expand any local variables passed as strings (e.g. PROJECT_OUTPUT_DIR)
 eval "SV_ARGS=\"${SV_ARGS}\""
 
-# Choose NUM_EXECUTORS = 2 * NUM_WORKERS
```

Comment:
I think Chris is currently having a fix for this behavior (i.e. formatting issue of the gcloud commands), see #4767

Reply:
Yes, I just merged that PR with a fix to `copy_sv_results.sh` that should make this change unnecessary.

Reply:
I think a change is still necessary, since run_whole_pipeline.sh executes first and needs to know the number of preemptible + normal workers to set the number of executors. I'm happy to change it to use the same solution you did in your PR.